src/test-utils/with-app-test-wrapper.js

```diff
@@ -0,0 +1,27 @@
+import { DataProvider } from '@dhis2/app-runtime'
+import PropTypes from 'prop-types'
+import React from 'react'
+import { MemoryRouter as Router, Route } from 'react-router-dom'
+import { QueryParamProvider } from 'use-query-params'
+import { CurrentItemProvider } from '../shared/index.js'
+import QueryClientWrapper from './query-client-wrapper.js'
+
+export const withAppTestWrapper = (
+    component,
+    initialUrl = '?dataSetId=BfMAe6Itzgt&orgUnitId=DiszpKrYNg8&periodId=202202'
```

Comment:
IMO this should be:

```suggestion
    initialUrl = ''
```

This special initial url should be moved to the validation results sidebar test as it's being used there only